site/source/tutorials/008_animations/demo/finished/zombies/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Can you upgrade these to beta5 (and make and required changes to the codes)?

Reply:
Also typescript to 2.6.1 (can just take a template from my outstanding PR for the beta5 upgrade).

Shouldn't need the animation types any more (after moving to beta5) :)